XFBindableStackLayout/Views/MainPage.xaml

```diff
@@ -7,11 +7,12 @@
     <ContentPage.BindingContext>
         <local:MainViewModel />
     </ContentPage.BindingContext>
-    <local:BindableStackLayout
+    <local:BindableStackLayout 
```

Comment:
If possible, it will make sense to add a sample of binding to `SelectedItem`.
It will require to implement a `INotifyPropertyChanged` on the VM level and add public property with getter and setter.